Watchman.Discord/Areas/Responses/Controllers/ResponsesController.cs

```diff
@@ -48,6 +49,14 @@ public async Task AddResponse(DiscordRequest request, Contexts contexts)
                 await messageService.SendResponse(x => x.ResponseAlreadyExists(contexts, onEvent));
                 return;
             }
+
+            var defaultResponse = await this._responsesService.GetResponseByOnEvent(onEvent, DomainResponse.DEFAULT_SERVER_ID);
+            if (defaultResponse.Message == message)
+            {
+                await messageService.SendResponse(x => x.ResponseHasBeenAdded(contexts, onEvent));
```

Comment:
tak teraz pomyślałem, że chyba lepiej wysyłać inny komunikat
typu: SameResponsesAlreadyExists
bo załóżmy że jakiś admin doda komendę która będzie identyczna do tej domyślnej
dostanie komunikat że pomyślnie dodano, więc uzna że tak już będzie
ale nagle zmieniamy tą domyślną komendę
w tym momencie zmienia się również na tym serwerze gdzie podobno "dodano odpowiedź", a w praktyce nie
Pewnie szansa na to jest bardzo mała, ale myślę że lepiej zwracać komunikat że taka odpowiedź jest już ustawiona